app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -51,6 +51,10 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
         setupToolbar()
         setupRecyclerView()
         presenter.loadChatRooms()
+        swipe_refresh_layout.setOnRefreshListener {
+            presenter.loadChatRooms()
+            swipe_refresh_layout.isRefreshing = false
```

Comment:
You should set `swipe_refresh_layout.isRefreshing = false` at `updateChatRooms()`

Reply:
Also we should or prevent the Swito to Refresh when using the search, or reapply the search query or clear the search.

Reply:
I thinked of setting swipe_refresh_layout.isRefreshing = false at updateChatRooms() but problem with that is we already have three-dot loading indicator, so it won't look good showing two loading indicator.

Will update PR soon for reapplying or clearing search